src/ansys/templates/python/common/{{cookiecutter.__project_name_slug}}/pyproject.toml

```
@@ -32,20 +32,18 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
 ]
-dependencies = [
-    "importlib-metadata >=4.0",
-]
+dependencies = []
 
 [project.optional-dependencies]
 tests = [
     "pytest==7.4.3",
     "pytest-cov==4.1.0",
 ]
 doc = [
-    "ansys-sphinx-theme==0.12.5",
-    "numpydoc==1.6.0",
-    "sphinx==7.2.6",
-    "sphinx-copybutton==0.5.2",
+    "ansys-sphinx-theme==0.15.2",
+    "numpydoc==1.5.0",
+    "sphinx==5.3.0",
+    "sphinx-copybutton==0.5.1",
```

```suggestion
    "numpydoc==1.7.0",",
    "sphinx==7.3.7",
    "sphinx-copybutton==0.5.2",
```
why are we  changing the dependencies here, since you upgraded the theme, we need to upgrade the other dependencies to use all the upgraded features

---

That sounds good! I missed upgrading them